articles/healthcare-apis/fhir/use-postman.md

```diff
@@ -100,6 +100,49 @@ You can examine the access token using online tools such as [https://jwt.ms](htt
 
 [ ![Screenshot of access token claims.](media/postman/postman-access-token-claims.png) ](media/postman/postman-access-token-claims.png#lightbox)
 
+## Get Microsoft Entra access token using user account with Authorization Code grant type
+
+You can get the Microsoft Entra access token using your Entra account credentials by following below listed steps
+
+1. As mentioned in the prerequisites section, you must be a member of Azure Entra Tenant and have required access.
+
+1. Ensure that you've configured the redirect URL, `https://oauth.pstmn.io/v1/callback`, for web platform in the client application registration.
+
+![callbackURL](media/use-postman/callbackurl.png)
+
+1. In the client application registration under API Permissions blade, Add **User_Impersonation** Delegated permission for **Azure Healthcare APIS** from **APIs my organization uses** tab.
+
+![App-reg-permissions](media/use-postman/app-reg-permissions.png)
+
+![App-reg-permissions-2](media/use-postman/app-reg-permissions-2.png)
+
+1. In the postman, select the **Authorization** tab of either a collection or a specific REST Call, select **Type** as OAuth 2.0 and under **Configure New Token** section, set below values 
+  - **Callback URL**: `https://oauth.pstmn.io/v1/callback`
+    
+  - **Auth URL**: `https://login.microsoftonline.com/{{tenantid}}/oauth2/v2.0/authorize`
+    
+  - **Access Token URL**: `https://login.microsoftonline.com/{{tenantid}}/oauth2/v2.0/token`
+    
+  - **Client ID**: Application client registration ID  
+    
+  - **Client Secret**: Application client registration secret 
+    
+  - **Scope**: `{{fhirurl}}/.default`
+    
+  - **Client Authentication**: Send client credentials in body
+    
+![Postman configuration](media/use-postman/postman-configuration.png)
+
+1. Click on **Get New Access Token** button at the bottom of page.
+
+1. You will be asked for User credentials for login.
+
+1. You will receive the token. Click on **Use Token.**
+
+1. Ensure Token is in the Authorization Header of the REST call.
+
+You can examine the access token using online tools such as [https://jwt.ms](https://jwt.ms). Select the Claims tab to see detailed descriptions for each claim in the token.
+
```

Comment:
After this is setup, how does customer validate they are getting capability statement. I request to move the section -"Connect to FHIR server" after this .

Reply:
Done in PR https://github.com/MicrosoftDocs/azure-docs-pr/pull/271776